src/js/InstitutionStatus.jsx

```diff
@@ -28,7 +28,7 @@ var InstitutionStatus = React.createClass({
     var submission = this.props.institution.currentSubmission;
     var submissionRoute = submission === 0 ? 1 : submission;
     var appLink = '/' + id + '/' + period + '/' + submissionRoute;
-    var statusLink = <Link to={appLink}>View filing status</Link>
+    var statusLink = <Link className="statusLink" to={appLink}>View filing status</Link>
```

Comment:
I don't think this class is necessary. I like the styling but I think its easier to solve it in other ways, e.g. wrapping it in a simple `<p>`.
_Same for others below._